### ISSUE TITLE
[TF2] Monoculus loses rimlight and phong when angry

### ISSUE THREAD

**Issue by Ashetf2** [ValveSoftware/Source-1-Games#4346]
Because in haunted_eyeball_angry.vmt, $phong values are commented and $rimlight is missing, when Monoculus is angry, it doesn't have phong/rimlight.

```
 	$nodecal 1

//	"$phong" "1"
//	"$phongexponent" "25"
	"$phongboost" "6"
	"$lightwarptexture" "models/player/pyro/pyro_lightwarp"
	"$phongfresnelranges"	"[1 1 1]"
	"$phongexponenttexture" "models\props_halloween\halloween_demoeye_brown_exp"
	"$basemapalphaphongmask" "1"
	"$phongalbedotint" "1"

//	"$envmap" "env_cubemap"
//	"$envmaptint" "[.25 .25 .25]"
```

Fix from https://github.com/agrastiOs/Ultimate-TF2-Visual-Fix-Pack

```
"VertexLitGeneric"
{
	"$baseTexture" "models\props_halloween\haunted_eyeball_angry"
	"$bumpmap" "models\props_halloween\haunted_eyeball_normal" 
	"$surfaceprop"	"flesh"
	"$one" "1"

	"$nodecal" "1"

	"$phong" "1"
	"$phongexponent" "10"
	"$phongboost" ".3"	
	"$lightwarptexture" "models/player/pyro/pyro_lightwarp"
	"$phongfresnelranges"	"[.1 1 8]"
	"$halflambert" "0"

	"$rimlight" "1"
	"$rimlightexponent" "4"
	"$rimlightboost" "2"

	// Cloaking
	"$cloakPassEnabled" "1"

	"Proxies"
	{
		"spy_invis"
		{
		}
	}
}
```

**Comment by condor00fr** [comment]
I see this fix comes from the tf2 visual fix pack, I really hope this mod ends up being added once it gets a proper release, the amout of work its devs put into it  in is fenomenal and really gives tf2 a new code of paint it desesperally needs right now.

**Comment by Chickenman456** [comment]
Does visual fix have its own issue? I would LOVE to see it fully implemented in the game

**Comment by condor00fr** [comment]
Yeah it has it's very own githud. However it's been a bit silent over the months. I sure hope the first official release gets out soon. They stated multiples times that as soon as it out and  that they finished promotional content for it (screenshots/video clips, ect) they would directy send the build over to Valve with explaination on what changed and  how to implement the fix. Trully hope this gets in, it's REALLY good.

**Comment by Ashetf2** [comment]
How it should look

![Image](https://github.com/user-attachments/assets/e0fe627d-2912-41b2-bcd7-91ac1e9b9bc9)

How it looks

![Image](https://github.com/user-attachments/assets/12cc991a-b8a4-44e2-9d27-203009d14fc1)

**Comment by Whurrhurr** [comment]
> Fix from https://github.com/agrastiOs/Ultimate-TF2-Visual-Fix-Pack
> 
> ```
> "VertexLitGeneric"
> {
> 	"$baseTexture" "models\props_halloween\haunted_eyeball_angry"
> 	"$bumpmap" "models\props_halloween\haunted_eyeball_normal" 
> 	"$surfaceprop"	"flesh"
> 	"$one" "1"
> 
> 	"$nodecal" "1"
> 
> 	"$phong" "1"
> 	"$phongexponent" "10"
> 	"$phongboost" ".3"	
> 	"$lightwarptexture" "models/player/pyro/pyro_lightwarp"
> 	"$phongfresnelranges"	"[.1 1 8]"
> 	"$halflambert" "0"
> 
> 	"$rimlight" "1"
> 	"$rimlightexponent" "4"
> 	"$rimlightboost" "2"
> 
> 	// Cloaking
> 	"$cloakPassEnabled" "1"
> 
> 	"Proxies"
> 	{
> 		"spy_invis"
> 		{
> 		}
> 	}
> }
> ```

so i made this, and i submitted it to the pack saying that fire details were omitted because you can't ignite monoculus, making the fire overlay pointess, and that's not really something you should do when fixing issues in vmts like what i did here, since someone could use a model that has monoculus with the angry texture used and then when igniting him, there's no fire overlay on the model. here's a zip where everything is consistent with haunted_eyeball

[Enraged Monoculus Fix.zip](https://github.com/user-attachments/files/28537895/Enraged.Monoculus.Fix.zip)
